Bottom: 373318c5562d8d193db0aed1a58b060956900e61
Top:    1bebf376438500f10ecffb619322820c989931a0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-13 19:29:40 +0100

Refresh of win32-fix-build-for-win32

---

diff --git a/src/hid/common/hidgl_shaders.c b/src/hid/common/hidgl_shaders.c
index 005d6f2..209dc7c 100644
--- a/src/hid/common/hidgl_shaders.c
+++ b/src/hid/common/hidgl_shaders.c
@@ -29,18 +29,24 @@
 #include <string.h>
 #include <stdio.h>
 
-#ifdef WIN32
-#   include <GL/gl.h>
-#   include <GL/glu.h>
-#   include "glext.h"
-#else
-/* The Linux OpenGL ABI 1.0 spec requires that we define
- * GL_GLEXT_PROTOTYPES before including gl.h or glx.h for extensions
- * in order to get prototypes:
- *   http://www.opengl.org/registry/ABI/
- */
+#ifndef WIN32
+    /* The Linux OpenGL ABI 1.0 spec requires that we define
+     * GL_GLEXT_PROTOTYPES before including gl.h or glx.h for extensions
+     * in order to get prototypes:
+     *   http://www.opengl.org/registry/ABI/
+     */
 #   define GL_GLEXT_PROTOTYPES 1
+#endif
+
+#ifdef HAVE_OPENGL_GL_H
+#   include <OpenGL/gl.h>
+#else
 #   include <GL/gl.h>
+#endif
+
+#ifdef HAVE_OPENGL_GLU_H
+#   include <OpenGL/glu.h>
+#else
 #   include <GL/glu.h>
 #endif
 
@@ -51,6 +57,8 @@
 #endif
 
 #ifdef WIN32
+#   include "glext.h"
+
 extern PFNGLGENBUFFERSPROC         glGenBuffers;
 extern PFNGLDELETEBUFFERSPROC      glDeleteBuffers;
 extern PFNGLBINDBUFFERPROC         glBindBuffer;
